### DISCUSSION TITLE
LinkedIn Showcase Pages are not detected as LinkedIn URI

### DISCUSSION THREAD

**Discussion by itrich** [community/community#46481] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Thank you for the new possibility to add up to four arbitrary social media links to a personal or organizational profiles. I've been long awaiting this feature and really happy about this change.

While updating the profile of the @SovereignCloudStack organization, I spotted that adding a link to a LinkedIn Showcase page (https://www.linkedin.com/showcase/sovereigncloudstack) is not correctly detected as a LinkedIn ressource:

![image](https://user-images.githubusercontent.com/29982899/216927333-89ebaca1-0524-4458-918b-d64675389cf1.png)

The result is a plain link to the Showcase page:

![image](https://user-images.githubusercontent.com/29982899/216927499-71017fd9-2530-4e7a-94aa-50f22d5b53a3.png)

**Comment by Anmol-Baranwal** [comment]
I don't believe it's a bug.
What if this feature is expressly designed to highlight your profile, in which case any link other than the direct profile link may be considered as a simple URL?
I'm not sure, but based on the feature's specifications, this could be the case.

**Reply by itrich** [reply]
I have to disagree. While a company page is correctly detected as a LinkedIn ressource, a showcase page is not.

As an owner of a GitHub organization, I'd expect that I can link both, a company as well as a showcase page.

Both page types are more or less the same, whereas a showcase page on LinkedIn is a page owned by a company, e.g. to promote a product or a subsidary. I guess this also applies to some Open Source projects which are hosted by a company, so this would be a helpful feature for some projects on GitHub.
